app/views/layouts/_sidebar.html.erb

```diff
@@ -6,11 +6,11 @@
   <li><%= link_to 'Home', welcome_index_path %></li>
 
   <!-- TODO - This should only display if a user has registered for events. The second part of this if statement doesn't work. -->
-  <% if user_signed_in? && !current_user&.user_events&.blank? %>
+  <% if user_signed_in? && current_user&.current_events&.present? %>
```

Comment:
Does present check for the existence of the array or it it contains items? I was having some issues with using 'present' to check on things before.